.eslintrc.js

```diff
@@ -0,0 +1,28 @@
+const os = require('os');
+const endLineStyle = os.platform() === 'win32'
```

Comment:
Will this introduce different line-endings depending on system?
Have you installed editorconfig for VS code?
https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig

Reply:
Line endings should always be LF.